tests/DataGrid/BetweenWriter/WriteOriginalTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Spiral Framework. Data Grid Bridge.
+ *
+ * @license MIT
+ * @author  Valentin Vintsukevich (vvval)
+ */
+
+declare(strict_types=1);
+
+namespace Spiral\Tests\DataGrid\BetweenWriter;
+
+use Spiral\DataGrid\Compiler;
+use Spiral\DataGrid\Specification\Filter;
+use Spiral\DataGrid\Specification\Value\StringValue;
+use Spiral\DataGrid\SpecificationInterface;
+use Spiral\DataGrid\Writer\OriginalBetweenWriter;
+use Spiral\DataGrid\Writer\QueryWriter;
+use Spiral\Tests\DataGrid\BaseTest;
+
+class WriteOriginalTest extends BaseTest
+{
+    public function testBetween(): void
+    {
+        $select = $this->compile(
+            $this->initQuery(),
+            new Filter\Between('field', [1, 2])
+        );
+
+        $this->assertEqualSQL(
+            'SELECT * FROM "users" WHERE "field" BETWEEN 1 AND 2',
+            $select
+        );
+
+        $select = $this->compile(
+            $this->initQuery(),
+            (new Filter\Between('field', new StringValue()))->withValue([1, 3])
+        );
+        $this->assertEqualSQL(
+            'SELECT * FROM "users" WHERE "field" BETWEEN \'1\' AND \'3\'',
+            $select
+        );
+    }
+
+    public function testValueBetween(): void
+    {
+        $select = $this->compile(
+            $this->initQuery(),
+            new Filter\ValueBetween(12, ['created', 'updated'])
+        );
+
+        $this->assertEqualSQL(
+            'SELECT * FROM "users" WHERE "12" BETWEEN \'created\' AND \'updated\'',
```

Comment:
expected to be
```php
'SELECT * FROM "users" WHERE 12 BETWEEN \'created\' AND \'updated\'',
```
caused by quoting first argument [here](#diff-7f776187a6b1e57b0fb10549186f1cf4R42)
```php
            return $source->where(
                $specification->getValue(),
                'BETWEEN',
                ...$specification->getExpression()
            );
```